docs/DeveloperGuide.md

```diff
@@ -2,36 +2,530 @@
 
 ## Acknowledgements
 
-{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
+1. Reference to AB-3 Developer Guide
+
```

Comment:
Nice work! I like the part of giving achnowledgements to appreciate the ideas from others.